### PR TITLE
Clarify state trigger functionality

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Clarify state trigger functionality

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation/trigger.markdown [hunk 1]
```diff
@@ -170,6 +171,16 @@ automation:
     to: "home"
 ```
 
+It's possible to give a list of frome_states or to_states:
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]:
```suggestion
It's possible to give a list of frome_states or to_states:

```

File: source/_docs/automation/trigger.markdown [hunk 2]
```diff
@@ -170,6 +171,16 @@ automation:
...
+```yaml
+automation:
+  trigger:
+    platform: state
+    entity_id: vacuum.test
+    from: "cleaning", "returning"
```
Comment (frenck) [inline source/_docs/automation/trigger.markdown]: Let's use actual YAML syntax as a good practice in our examples.

```yaml
    from:
      - "cleaning"
      - "returning"
```

or

```yaml
    from: ["cleaning", "returning"]
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/automation/trigger.markdown (hunk 2 above)
Comment (emontnemery) [reply source/_docs/automation/trigger.markdown]: Fixed.

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery 👍

**Pull request merged**